Gemfile

```diff
@@ -1,17 +1,21 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-ruby "2.2.3"
+ruby '2.2.5'
 
 gem 'sinatra'
 gem 'json'
 gem 'octokit'
 
-group :test do 
+group :development do
+  gem 'rubocop'
+  gem 'guard'
+  gem 'guard-rubocop'
```

Comment:
in theory, you could just have `guard-rubocop` - both the other deps would come from that